Dockerfile.apk

```diff
@@ -3,11 +3,11 @@ FROM alpine:3.16.0
 
 LABEL maintainer="Kong Docker Maintainers <docker@konghq.com> (@team-gateway-bot)"
 
-ARG KONG_VERSION=2.8.1
+ARG KONG_VERSION=3.0.0
```

Comment:
👍 This will help some questions we have been getting. Will this also require an update to [KBT](https://github.com/Kong/kong-build-tools/blob/a93e5cb39a2c7fb78aaba9eea6b7ecf0c2ddac2f/Makefile#L77)?

Reply:
I think so. cc @hutchic ^